src/test/java/racingcar/CarNameTest.java

```diff
@@ -0,0 +1,45 @@
+package racingcar;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatCode;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import racingcar.domain.CarName;
+
+class CarNameTest {
+    private static final String validName = "name";
+    private static final String overLengthName = "isOverLength";
+    private static final String emptyName = "";
+    private static final String spaceBarName = "    ";
+
+    @Test
+    @DisplayName("::정상적인 이름이 입력될 경우 예외가 발생하지 않는다")
+    void testValidName() {
+        assertThatCode(() -> new CarName(validName)).doesNotThrowAnyException();
+    }
+
+    @Test
+    @DisplayName("::5자리를 넘어가는 이름이 입력될 경우 예외가 발생한다")
+    void testOverLength() {
+        assertThatThrownBy(() -> new CarName(overLengthName))
+                .isInstanceOf(IllegalArgumentException.class)
+                .hasMessage("자동차 이름은 5자 까지만 입력할 수 있습니다.");
```

Comment:
```suggestion
                .hasMessage(OUT_OF_STANDARD.getMessage())
```

위에서 정의한 Enum의 메세지를 호출해서 사용하면, 자동차 이름의 제약조건이 변하더라도, 계속적으로 활용할 수 있을 것 같아요!